source/guides/community-chat.rst

```diff
@@ -0,0 +1,25 @@
+Mattermost Community Chat Server
+================================
+
+Mattermost hosts a chat server at `community.mattermost.com <https://community.mattermost.com/login>`__ to connect our team with our larger contributor and user community. This server is open to everyone, and we welcome anyone who wants to participate in our community to sign up.
```

Comment:
Is "chat" the right wording in this case? Maybe "collaboration server" could work?

Reply:
I updated the language in this.